README.md

```diff
@@ -1,10 +1,13 @@
-# Parallel Programming on Discovery
-This repository contains material for HPC Parallel Programming on Discovery
+# HPC & Parallel Computing
+This repository contains material for HPC Parallel Computing on Discovery
 training modules.
 
 ## What's in each training sub-folder?
-* Training PowerPoint slides
-* Exercise folders
+* Exercise_1 - OpenMP threads example
+* Exercise_2 - MPI with Python example (mpi4py)
+* Exercise_3 - Slurm job array example
+* Exercise_4 - Speendup, efficiency and weak scaling examples. 
```

Comment:
@marianalevineu , as @joseph-cooke pointed out, it should be Speedup and not Speendup.

Reply:
thanks @joseph-cooke  @manasvita  the typo was fixed.